From 8206b824fa3cba5cbc7ddec0a1ccec7828f37dc5 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Sat, 26 Mar 2016 09:32:51 -0400
Subject: Make gdk_visual_get_*_pixel_details work again

These functions are supposed to return the numbers of consecutive
1 bits in each components mask as precision. However, due to a
copy-paste mistake when this code was moved around in
commit 70d689cddda0dc616af97e8ed047d0c0acf7c7a6, the precision
was always reported as zero. This affects only a few applications
that directly set window background on X11 windows, such as emacs.

https://bugzilla.gnome.org/show_bug.cgi?id=764210
---
 gdk/gdkvisual.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/gdk/gdkvisual.c b/gdk/gdkvisual.c
index d580c6b..356ea19 100644
--- a/gdk/gdkvisual.c
+++ b/gdk/gdkvisual.c
@@ -371,7 +371,6 @@ gdk_visual_get_pixel_details (GdkVisual *visual,
           m >>= 1;
         }
 
-      m = pixel_mask;
       while (m & 0x1)
         {
           p++;
-- 
cgit v0.12

